UI/cypress/e2e/results.js

```diff
@@ -188,3 +188,17 @@ Then("results are updated after filters are cleared", () => {
 Then("the postcode search parameter remains", () => {
   cy.get('[data-testid="postcode-input-box"]').should("have.value", "sk11eb");
 });
+
+Then(
+  "the number of tuition partners displayed matches the displayed count",
+  () => {
+    let countOfElements = 0;
+    cy.get('[data-testid="results-list-item"]').then(($elements) => {
+      countOfElements = $elements.length;
+      cy.get('[data-testid="result-count"]')
+        .invoke("text")
+        .then(parseInt)
+        .should("equal", countOfElements);
```

Comment:
This is a really good way of asserting the count matches the number of results.  Much better than my idea of hard coding the count! 💯